var/spack/repos/builtin/packages/py-backcall/package.py

```diff
@@ -0,0 +1,35 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyBackcall(PythonPackage):
+    """Specifications for callback functions passed in to an API"""
+
+    homepage = "https://github.com/takluyver/backcall"
+    url = "https://pypi.io/packages/source/b/backcall/backcall-0.1.0.tar.gz"
+
+    version('0.1.0', '87ce0c7839808e6a3427d57df6a792e7')
+    depends_on('python')
```

Comment:
This is already present in the `PythonPackage` base class, so you don't need this line.